app/assets/scripts/components/results-panel.js

```diff
@@ -97,9 +97,9 @@ const Results = React.createClass({
                 <h3 className='subtitle results__subtitle'>{t('Exposure')}</h3>
                 <dl className='stats'>
                   <dt className='stat__attribute'>{t('GDP')}</dt>
-                  <dd className='stat__value unimplemented'>${shortenNumber(d.EX_GD, 2, false)}</dd>
+                  <dd className='stat__value unimplemented'>$ {d.EX_GD === undefined ? ' -' : shortenNumber(d.EX_GD, 2, false)}</dd>
```

Comment:
@nbumbarger The revision doesn't seem to work. It was super picky about the way I tested the value for being undefined.